### ISSUE TITLE
kornia.geometry.epipolar.essential_from_Rt fails with volumetric data!

### ISSUE THREAD

**Issue by nitaifingerhut** [kornia/kornia#1527]
### Describe the bug

`kornia.geometry.epipolar.essential_from_Rt` has a problem with volumetric data.
assume we work with volumetric data in format `BTCHW` , so the input shapes are:
`R1=(B, T, 3, 3), t1=(B, T, 3, 1), R2=(B, T, 3, 3), t3=(B, T, 3, 1)`
after line:
`R, t = relative_camera_motion(R1, t1, R2, t2)`
the shapes of R, t will be:
`R=(B, T, 3, 3), t=(B, T, 3, 1)`
Then, it calls
`Tx = numeric.cross_product_matrix(t[..., 0])`
which expect a 2D input of shape `(B, 3)` , however the shape of `t[..., 0]`  is `(B, T, 3)`  !!!

### Reproduction steps

```bash
import torch

from kornia.geometry.epipolar import essential_from_Rt

R1 = torch.rand(4, 2, 3, 3)
t1 = torch.rand(4, 2, 3, 1)
R2 = torch.rand(4, 2, 3, 3)
t2 = torch.rand(4, 2, 3, 1)
e = essential_from_Rt(R1, t1, R2, t2)
```

### Expected behavior

working properly with volumetric data!

### Environment

```shell
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```
- PyTorch Version (e.g., 1.0):
- OS (e.g., Linux):
- How you installed PyTorch (`conda`, `pip`, source):
- Build command you used (if compiling from source):
- Python version:
- CUDA/cuDNN version:
- GPU models and configuration:
- Any other relevant information:
```


### Additional context

_No response_

**Comment by nitaifingerhut** [comment]
IMHO, the fix is to from `kornia.geometry.epipolar.numeric`:

`
def cross_product_matrix(x: torch.Tensor) -> torch.Tensor:
    r"""Returns the cross_product_matrix symmetric matrix of a vector.

    Args:
        x: The input vector to construct the matrix in the shape :math:`(*, 3)`.

    Returns:
        The constructed cross_product_matrix symmetric matrix with shape :math:`(*, 3, 3)`.

    """
    # get vector compononens
    x0 = x[..., 0]
    x1 = x[..., 1]
    x2 = x[..., 2]

    # construct the matrix, reshape to 3x3 and return
    zeros = torch.zeros_like(x0)
    cross_product_matrix_flat = torch.stack([zeros, -x2, x1, x2, zeros, -x0, -x1, x0, zeros], dim=-1)
    shape_ = x.shape[:-1] + (3, 3)
    return cross_product_matrix_flat.view(*shape_)
`
